routes/index.js

```diff
@@ -15,6 +16,7 @@ router.use('/purchasesorders', purchasesOrders);
 router.use('/products', products);
 router.use('/activitiesproducts', activitiesProducts);
 router.use('/login', login);
+router.use('/usersactivities', usersActivities);
```

Comment:
/useractivities/:userId/activities isn't really REST

it looks like you want to retrieve the activities for a user, so it should be: /users/:userId/activities